Gopkg.toml

```diff
@@ -104,6 +104,10 @@ required = [
   name = "github.com/knative/test-infra"
   non-go = false
 
+[[override]]
+  name = "github.com/kelseyhightower/envconfig"
+  branch = "master"
```

Comment:
Should we lock this to the last release?

Reply:
Used last release version ```v1.4.0```